assembly/assembly-machine-war/src/main/java/org/eclipse/che/wsagent/server/WsAgentServletModule.java

```diff
@@ -13,19 +13,24 @@
 import com.google.common.collect.ImmutableMap;
 import com.google.inject.servlet.ServletModule;
 
-import org.eclipse.che.env.local.server.SingleEnvironmentFilter;
+import org.eclipse.che.api.local.SingleEnvironmentFilter;
 import org.eclipse.che.inject.DynaModule;
 import org.everrest.guice.servlet.GuiceEverrestServlet;
 import org.everrest.websockets.WSConnectionTracker;
 
+import javax.inject.Singleton;
+
 /** @author andrew00x */
 @DynaModule
-public class ApiServletModule extends ServletModule {
+public class WsAgentServletModule extends ServletModule {
     @Override
     protected void configureServlets() {
         getServletContext().addListener(new WSConnectionTracker());
+
+        bind(SingleEnvironmentFilter.class).in(Singleton.class);
```

Comment:
I think you can remove `.in(Singleton.class)` because class had @Singleton annotation